### ISSUE TITLE
CI is sloooow

### ISSUE THREAD

**Issue by axelboc** [mxcube/mxcube3#1424]
You've probably noticed it. Not sure if it's a temporary slow down or if it's something to do with the recent change to test all supported Python versions. https://github.com/mxcube/mxcubeweb/actions/runs/11069955541

![image](https://github.com/user-attachments/assets/973f9fa7-4a9e-4aca-9aed-98940a721100)

**Comment by marcus-oscarsson** [comment]
I think its temporary, it was quite quick earlier today. The pages build took forever though.

**Comment by fabcor-maxiv** [comment]
Nothing obvious on https://www.githubstatus.com/

**Comment by marcus-oscarsson** [comment]
Looks like the actions are stuck on
```
micromamba list
  /home/runner/micromamba-bin/micromamba list -r /home/runner/micromamba -n mxcubeweb
```

A bit strange, considering that is just listing the installed packages. It does not seem to matter if we are only running one python version or several simultaneously

**Comment by fabcor-maxiv** [comment]
There was a new major release of mamba and micromamba. Versions 2.0.0 for both. Maybe something broke.

We can try:
```yaml
- uses: mamba-org/setup-micromamba@v1
  with:
    micromamba-version: '1.5.10-0' # any version from https://github.com/mamba-org/micromamba-releases
```

**Comment by fabcor-maxiv** [comment]
https://github.com/mamba-org/setup-micromamba/issues/225

**Comment by marcus-oscarsson** [comment]
I think thats it !

**Comment by fabcor-maxiv** [comment]
I am a bit reluctant to make a fix for it that would involve pinning the version of micromamba...

* If we are lucky, there will be a fix in micromamba (or the `setup-micromamba` action) soon enough.
* Do we really want to pin the version of micromamba? Would dependabot help us keep the pinned version up to date?

**Comment by fabcor-maxiv** [comment]
For info:

Looks like in principle the root issue has been fixed and the fixed version has been released (in version 2.0.1):
* <https://github.com/mamba-org/mamba/pull/3490>
* <https://github.com/mamba-org/mamba/releases/tag/2024.09.30>
* <https://github.com/mamba-org/setup-micromamba/issues/225#event-14490786159>

So I assume that in principle we could remove the pin (I have not tried it). On the other hand, maybe it is not a bad idea to wait a bit longer until micromamba v2 is more stable in general.
